browser/BUILD.gn

```diff
@@ -107,23 +108,36 @@ source_set("browser_process") {
     ":sparkle_buildflags",
     ":version_info",
     "autoplay",
+    "brave_stats:stats_updater",
+    "browsing_data",
+    "browsing_data",
+    "browsing_data",
     "content_settings",
     "download",
     "farbling",
     "net",
+    "ntp_background_images",
+    "ntp_background_images",
+    "ntp_background_images",
     "profiles",
     "renderer_context_menu",
+    "search",
+    "search",
+    "themes",
+    "themes",
+    "themes",
     "ui",
     "//base",
     "//brave/browser/net",
     "//brave/browser/notifications",
     "//brave/browser/tor:buildflags",
     "//brave/browser/translate/buildflags",
+    "//brave/chromium_src:browser",
+    "//brave/chromium_src:browser",
+    "//brave/chromium_src:browser",
```

Comment:
Is there any reason whey this deps is added here?
`//brave/chromium_src:browser` is empty target now.
and it's addede repeatedly?